src/app/workflow/view/view.component.html

```diff
@@ -17,6 +17,9 @@
 <button type="button" mat-button color="accent" class="btn-block" (click)="showVersionModal()">
   {{ isPublic || !canWrite || (entryType$ | async) === EntryType.Service ? 'View' : 'Edit' }}
 </button>
+<button type="button" mat-button color="accent" class="btn-block" *ngIf="!version.frozen" (click)="snapshotVersion()">
```

Comment:
At story time, didn't we say this was sort of a big deal, and that there should be a confirmation dialog before proceeding -- otherwise you might accidentally click/freeze something without meaning to? And/or was that for Zenodo?

Also, I forget, did we agree to go with Snapshot as the public facing name? API is calling it frozen, and TRS discussions are apparently using the word immutable.

Reply:
1. I will add a dialog.

2. I am +0 on whatever name we choose but figured we might go with frozen or read-only. Can change as necessary but will become more of a hassle once we write docs.

Reply:
I am +0 on name too... Maybe Denis will have a stronger sense after Bosc.